app/partials/coinify/checkout.pug

```diff
@@ -0,0 +1,88 @@
+bc-tabs(tab="tabs.selectedTab" tab-options="tabs.options" on-select="tabs.select($tab)" badge="trades.pending.length")
+  .sfox-logo
+    span.mrs.type-sm Powered By
+    a(href="https://www.coinify.com/" target="_blank" rel="noopener noreferrer")
+      img(src="img/coinify-logo.svg")
+.buy-sell.mbvl.mh-15
+  .flex-row.flex-column-tablet.flex-column-mobile(ng-show="tabs.selectedTab === 'BUY_BITCOIN'")
+    //- BUY
+    .width-50.border-desktop.pv-30.ph-30.no-padding-mobile
+      exchange-checkout(
+        fiat="fiat"
+        type="'Buy'"
+        conversion="100"
+        trading="buying"
+        limits="buyLimits"
+        provider="'Coinify'"
+        fiat-options="fiatOptions"
+        buy-success="buyHandler(quote)"
```

Comment:
Can probably improve the name of this binding to make it more clear regarding what it is and what it does.